### ISSUE TITLE
Native Loop doesn't work

### ISSUE THREAD

**Issue by woganmay** [alessandro-pezzato/omxdirector#4]
I'm using the latest build of omxplayer on my pi:

    Build date: Fri, 24 Oct 2014 00:48:49 +0000
    Version: 98982df [master]

It supports the native `--loop` switch. When I leave `enableNativeLoop()` off the `require` statement, then the fallback loop mechanism comes into play: omxdirector kills and respawns threads normally. If I switch it ON, then omxplayer just keeps repeating the same file, despite the fact that I tried playing it with an array of files.

I think the documentation should indicate that enableNativeLoop() will only loop one file, or the first file in the array you pass to `.play()`.

**Comment by alessandro-pezzato** [comment]
You're right, I think this is a bug. If I remember correctly, native loop can only loop a single file (maybe even files with exactly the same format/resolution), but the --loop flag when native loop is disabled loops all the files in the list. So I guess the correct behavior is not using native loop when more than one file are in the list. What do you think?

**Comment by kkcy** [comment]
Hi, i'm using 02 Jan 2016 version of omxplayer. I think there's a newer version out there, but Is the nativeloop still only enables a single file to be loop? That's a shame cause I really want a seamless multiple file playing experience.

**Comment by alepez** [comment]
AFAIK the seamless loop is possible only for the same file. Technically
it's possible for files with same encoding and size, but you have to use a
forked omxplayer (i don't remember where you can find it)

Il giorno Lun 21 Mar 2016 13:54 kkcy <notifications@github.com> ha scritto:

> Hi, i'm using 02 Jan 2016 version of omxplayer. I think there's a newer
> version out there, but Is the nativeloop still only enables a single file
> to be loop? That's a shame cause I really want a seamless multiple file
> playing experience.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/alepez/omxdirector/issues/4#issuecomment-199258501>
>
-- 
Alessandro Pezzato
